lib/abci/handlers/deliverTxHandlerFactory.js

```diff
@@ -39,9 +54,31 @@ function deliverTxHandlerFactory(
 
     await transactionalDpp.stateTransition.apply(stateTransition);
 
-    // Save data contracts in order to create databases for documents on block commit
-    if (stateTransition.getType() === stateTransitionTypes.DATA_CONTRACT_CREATE) {
-      blockExecutionState.addDataContract(stateTransition.getDataContract());
+    switch (stateTransition.getType()) {
+      case stateTransitionTypes.DATA_CONTRACT_CREATE: {
+        const dataContract = stateTransition.getDataContract();
+
+        // Save data contracts in order to create databases for documents on block commit
+        blockExecutionState.addDataContract(dataContract);
+
+        logger.debug(`Created data contract with id: ${dataContract.getId()}`);
```

Comment:
I would put the entity type to the beginning of the line. It will be easier to scan: "Data Contract created   ${dataContract.getId()}"